### PR TITLE
unit test AdminOrderController

### PR DESCRIPTION
**Pull request by ariesh2t**
## Related Tickets:
- [#47455](https://edu-redmine.sun-asterisk.vn/projects/hn_oe49-shoe-store/issues/47455)

## WHAT this PR does?
- Viết unit test cho AdminOrderController

## Notes Impacted Areas:
- N/A.

## Attachments:
https://user-images.githubusercontent.com/61032909/159636657-b5f81b03-3df4-45c5-a283-b669652fce01.png

### CODE REVIEW

**Review by NgocAnhDT — APPROVED** [review]

**Comment by ariesh2t** [comment]
ready

File: app/Repositories/Order/OrderRepository.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+namespace App\Repositories\Order;
+
+use App\Repositories\BaseRepository;
+
+class OrderRepository extends BaseRepository implements OrderRepositoryInterface
+{
+    //lấy model tương ứng
+    public function getModel()
+    {
+        return \App\Models\Order::class;
```
Comment (huyenttm-0901) [inline app/Repositories/Order/OrderRepository.php:11]:
```suggestion
        return Order::class;
```

File: app/Repositories/Order/OrderRepository.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        return $this->model->with('user', 'products', 'orderStatus')
+            ->orderBy('order_status_id')
+            ->orderBy('created_at', 'DESC')
+            ->paginate(config('paginate.pagination'));
+    }
+
+    public function relation($id)
+    {
+        $order = $this->find($id);
+        return $order->products;
```
Comment (huyenttm-0901) [inline app/Repositories/Order/OrderRepository.php:25]: convention

File: app/Repositories/OrderStatus/OrderStatusRepository.php [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+namespace App\Repositories\OrderStatus;
+
+use App\Repositories\BaseRepository;
+
+class OrderStatusRepository extends BaseRepository implements OrderStatusRepositoryInterface
+{
+    //lấy model tương ứng
+    public function getModel()
+    {
+        return \App\Models\OrderStatus::class;
```
Comment (huyenttm-0901) [inline app/Repositories/OrderStatus/OrderStatusRepository.php:11]: tương tự

File: tests/Unit/Http/Controller/AdminOrderControllerTest.php [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+        $this->mockOrderRepo->shouldReceive('getOrder')->once();
+        
+        $view = $this->controller->index();
+
+        $this->assertEquals('admins.orders.index', $view->getName());
+        $this->assertArrayHasKey('orders', $view->getData());
+    }
+
+    public function testShowReturnView()
+    {
```
Comment (huyenttm-0901) [inline tests/Unit/Http/Controller/AdminOrderControllerTest.php:62]: test case ko tìm thấy order nữa em. nếu hàm find trả về null thì sao?

**Comment by ariesh2t** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**